Python/Product/PythonTools/PythonTools/Debugger/DebugAdapter/DebugAdapterLauncher.cs

```diff
@@ -51,8 +52,8 @@ public sealed class DebugAdapterLauncher : IAdapterLauncher {
             var debugLaunchInfo = (DebugLaunchInfo)_debugInfo;
             var ptvsdAdapterDirectory = Path.GetDirectoryName(PythonToolsInstallPath.GetFile("Packages\\ptvsd\\adapter\\__init__.py"));
 
-            var targetProcess = new DebugAdapterProcess(targetInterop, ptvsdAdapterDirectory);
-            return targetProcess.StartProcess(debugLaunchInfo.InterpreterPathAndArguments.FirstOrDefault(), debugLaunchInfo.WebPageUrl);
+            var targetProcess = new DebugAdapterProcess(_adapterHostContext, targetInterop, ptvsdAdapterDirectory);
```

Comment:
could add a check for null on _adapterHostContext, to make sure they called initialize.